common/traits/000_wc_leadership_traits.txt

```diff
@@ -0,0 +1,218 @@
+# Written by: DukeJeans
+# Created: 1/22/2019
+# Modified: 3/28/2019
+#Leadership traits
+
+### Warcraft ones ###
+archmagus_leader = {
+	leader = yes
+	random = no
+	customizer = no
+
+	command_modifier = {
+		days_of_supply = 10
+		actual_light_infantry = 0.2
+		archmages = 0.2
+	}
+}
+
+marksman_captain_leader = {
+    leader = yes
+
+    command_modifier = {
+		archers = 0.30
+    }
+}
+
+aerial_raider_leader = {
+    leader = yes
+
+    command_modifier = {
+        knights = 0.20
+		pursue = 0.10
+    }
+}
+
+artillery_engineer_leader = {
+    leader = yes
+
+    command_modifier = {
+        siege_machines = 0.20
+        war_machines = 0.20
+    }
+
+	siege_speed = 0.15
+}
+
+taskmaster_leader = {
+    leader = yes
+	random = no
+
+    command_modifier = {
+        thralls = 0.20
+		speed = 0.10
+		supply_limit = 3
+    }
+}
+
+warpriest_leader = {
+    leader = yes
+	random = no
+	customizer = no
+
+	attrition = -0.10
+
+    command_modifier = {
+        morale_defence = 0.10
+		religious_enemy = 0.10
+        # max_attrition = -0.50
+        # days_of_supply = 30
+    }
+}
+
+offensive_caster_leader = {
+    leader = yes
+	random = no
+	customizer = no
+
+    command_modifier = {
+		actual_light_infantry = 0.1
+        damage = 0.10
+		morale_offence = 0.10
+    }
+}
+
+defensive_caster_leader = {
+    leader = yes
+	random = no
+	customizer = no
+
+	command_modifier = {
+		actual_light_infantry = 0.1
+        defence = 0.10
+		retreat = 0.25
+    }
+}
+
+defiler_leader = {
+    leader = yes
+	trait_effect_capture_commanders = 1
+	random = no
+	customizer = no
+
+    command_modifier = {
+        morale_offence = 0.30
+		pursue = 0.20
+    }
+
+	is_visible = {
+		FROM = {
+			ROOT = {
+				OR = {
+					prev_is_visible_warlock_to_this_trigger = yes
+					prev_is_visible_necromancer_to_this_trigger = yes
+					prev_is_visible_shadow_priest_to_this_trigger = yes
+				}
+			}
+		}
+	}
+}
+
+### Vanilla ones ###
+
+light_foot_leader = {
+	leader = yes
+
+	command_modifier = {
+		actual_heavy_infantry = 0.20
```

Comment:
Now it can be changed to "light_infantry", trait loc can be changed to "Support Specialist" or something like that